p2p/protocol.py

```diff
@@ -123,6 +126,27 @@ def encode(self, data: _DecodedMsgType) -> Tuple[bytes, bytes]:
         return header, body
 
 
+TCommandPayload = TypeVar('TCommandPayload', bound=_DecodedMsgType)
```

Comment:
With this class named `Request` the term payload implies something that we are delivering.  However, I believe that `TCommandPayload` is referring to the response payload.  Thoughts on naming change?

- `TResponsePayload`?

Reply:
This is actually the outgoing one, but we'll end up with much clearer `TRequestPayload` and `TResponsePayload`.